Grammar, Spelling & 'Sara' Refresh

DeveloperWOW64 opened the pull request:
## Description
Fixed grammar and spelling errors and changed slightly misunderstandable phrases, as well as obviously, 'sara':
Changed directories, date and time on sara's pc, as well as adding a couple of easter eggs (!) and generally refreshing things.

## Motivation and Context
Part required so people can easily read and understand the Website, as well as the refresh and easter eggs, all of which are unnecessary, more personal preference.

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/71598437/158074791-cb61e772-f3c0-4411-a3d4-0ba2f5b5935c.png)

TheOnlyTails commented on src/data/features.ts:
```diff
@@ -97,9 +97,9 @@ export const previewFiles = [
 		name: "Files v3",
 		icon: "/ui/icons/folder.png",
 		items: 16,
-		added: "Wednesday, October 6, 2021 42:69 AM",
-		modified: "Thursday, January 1, 1970 00:00",
```
These two were very intentional easter eggs.

DeveloperWOW64 replied:
Okay, I'll return that to it's original state. However, can I spread these Easter eggs out, not have them only on one file; As in, I'll put the 42:69 time on waves.png, and leave the '1970 bug' on Files v3. Also, where is the 'C:\Users\Sara\Documents\index.html' stored? I'd like to add a link to the 'Look at this cool button' to the files discord, if possible.

DeveloperWOW64 commented:
> A lot of these are less typo fixes and more personal preference (e.g. changing the tag colors, removing easter eggs, etc).

Yeah, I just thought it would be good to give it a little refresh.

Also, about easter eggs, can I spread them out, not have them only on one file? As in, I'll put the 42:69 time on waves.png, and leave the '1970 bug' on Files v3. Also, where is the 'C:\Users\Sara\Documents\index.html' stored? I'd like to add a link to the 'Look at this cool button' to the files discord, if possible.

TheOnlyTails commented:
> Also, where is the 'C:\Users\Sara\Documents\index.html' stored?

https://github.com/files-community/Website/blob/main/static/preview-samples/index.html

DeveloperWOW64 commented:
PS: I just added another easter egg:
![image](https://user-images.githubusercontent.com/71598437/158236935-ea994111-f224-49a7-998a-8fabd003dca5.png)
Also, I moved the 42:69 time to waves.png and restored the 1970 date

yaichenbaum commented on src/data/features.ts:
```diff
@@ -97,9 +97,9 @@ export const previewFiles = [
-		path: "C:\\Users\\Sara\\Desktop\\Files v3"
+		added: "Wednesday, October 6, 2021 9:34 PM",
+		modified: "Thursday, January 1, 1970 00:00 PM",
+		path: "C:\\Users\\Sara\\source\\repos\\Files v3"
```
Why the change?

DeveloperWOW64 replied:
VS's default repo folder is %userprofile%\source\repos and obvs files is a vs sln so I thought it would be better to have the project in there rather than Desktop.

yaichenbaum commented on src/data/features.ts:
```diff
@@ -114,24 +114,24 @@ export const previewFiles = [
-		path: "C:\\Users\\Sara\\Pictures\\waves.png"
+		added: "Sunday, August 1, 2021 42:69 AM",
+		modified: "Monday, August 9, 2021 12:34 PM",
+		path: "C:\\Users\\Sara\\OneDrive\\Pictures\\waves.png"
```
Can you revert the changes? The paths are fine as is.

DeveloperWOW64 replied:
In pictures, the 'Sara' uses OneDrive, so I thought it would be appropriate to put that in as well.

DeveloperWOW64 commented on static/preview-samples/index.html:
```diff
@@ -9,6 +9,8 @@
 <body>
 <h1>Hello World!</h1>
 <p>This is my first website.</p>
+<a href="https://discord.gg/files" target="_blank" rel="noopener noreferrer">
```
It's an Easter Egg.

DeveloperWOW64 commented on src/data/features.ts:
```diff
@@ -114,24 +114,24 @@ export const previewFiles = [
 		added: "Wednesday, July 14, 2021 8:47 PM",
-		modified: "Saturday, July 31, 2021 4:32 AM",
-		path: "C:\\Users\\Sara\\Documents\\index.html"
+		modified: "Saturday, July 31, 2021 6:32 AM",
```
Done.

yaichenbaum commented:
Closing due to inactivity
